client/src/Component/views/Methods/index.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
+import Header from '../../Common/Header';
+import Footer from '../../Common/Footer';
 
 const Methods = () => {
   return (
     <div>
+      <Header titleText="Selecet your methods" />
```

Comment:
Same problem as above in the Play page:
![Screen Shot 2019-04-19 at 3 15 48 PM](https://user-images.githubusercontent.com/9094166/56428425-38068780-62b7-11e9-99dd-19964db45272.png)
I think if you include the prop with a type of null like Ali suggested it should fix the errors!
``` <Header titleText="Choose your game play:" headerImg={null} />```